hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -1169,78 +1173,61 @@ private void handleOverReplicatedContainer(final ContainerInfo container,
         }
       }
       eligibleReplicas.removeAll(unhealthyReplicas);
-
-      excess -= handleMoveIfNeeded(container, eligibleReplicas);
-
       removeExcessReplicasIfNeeded(excess, container, eligibleReplicas);
     }
   }
 
   /**
-   * if the container is in inflightMove, handle move if needed.
+   * if the container is in inflightMove, handle move.
    *
    * @param cif ContainerInfo
-   * @param eligibleReplicas An list of replicas, which may have excess replicas
-   * @return minus how many replica is removed through sending delete command
+   * @param replicaSet An Set of replicas, which may have excess replicas
    */
-  private int handleMoveIfNeeded(final ContainerInfo cif,
-                   final List<ContainerReplica> eligibleReplicas) {
-    int minus = 0;
+  private void handleMove(final ContainerInfo cif,
```

Comment:
This function assumes replication has completed. It would be good to document that.

Reply:
It might be better to rename this function to sth like deleteSrcDnForMove.

Reply:
yes , i will do this , thanks for the review